client/packages/system/src/Clinician/ClinicianSearchInput.tsx

```diff
@@ -21,31 +28,75 @@ interface ClinicianSearchInputProps {
   disabled?: boolean;
   fullWidth?: boolean;
   allowCreate?: boolean;
-  mountCreateModalAsSidePanel?: boolean;
+  mountSlidePanel?: boolean;
 }
 
-export const ClinicianSearchInput: FC<ClinicianSearchInputProps> = ({
+export const ClinicianSearchInput = ({
   onChange,
   width = 250,
   clinicianValue,
   disabled,
   fullWidth,
   allowCreate,
-  mountCreateModalAsSidePanel = false,
-}) => {
+  mountSlidePanel = false,
+}: ClinicianSearchInputProps) => {
   const t = useTranslation();
-  const [modalOpen, setModalOpen] = useState(false);
-  const { data, refetch } = useClinicians.document.list({});
+  const theme = useTheme();
   const { getLocalisedFullName } = useIntlUtils();
+  const { error, success } = useNotification();
+  const { data, refetch } = useClinicians.document.list({});
+  const { isSaving, draft, updateDraft, isValid, save, clear } =
+    useCreateClinician();
+
   const clinicians: ClinicianFragment[] = data?.nodes ?? [];
-  const theme = useTheme();
+  const [modalOpen, setModalOpen] = useState(false);
+  const [sliderOpen, setSliderOpen] = useState(false);
 
   const asOption = (clinician: Clinician): ClinicianAutocompleteOption => ({
     label: getLocalisedFullName(clinician.firstName, clinician.lastName),
     value: clinician,
     id: clinician.id,
   });
 
+  const handleClinicianClose = async (clinicianId?: string) => {
```

Comment:
Pulled most things out to here so that it can be passed onto whatever child is being rendered